src/types/ellipse.js

```diff
@@ -18,6 +18,11 @@ export default class EllipseAnnotation extends BoxAnnotation {
     ctx.strokeStyle = options.borderColor;
     ctx.fillStyle = options.backgroundColor;
 
+    if (ctx.setLineDash) {
```

Comment:
These if's can be removed, it is for some old IE that is not supported anymore.

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash#browser_compatibility

Reply:
Removed everywhere. It was previously used in the line annotation and I have removed from there as well